client/layout/masterbar/oauth-client-woo.scss

```diff
@@ -0,0 +1,158 @@
+.masterbar__oauth-client-user-nav {
+	float: right;
+}
+
+.masterbar__oauth-client-wpcc-sign-in {
+	display: block;
+	color: #fff;
+	background-color: rgba(255, 255, 255, 0.15);
+	padding: 3px;
+	line-height: 100%;
+	position: relative;
+	top: 6px;
+	border-radius: 3px;
+	margin-right: 5px;
+}
+.masterbar__oauth-client-wpcc-sign-in:after {
+	content: " ";
+	display: block;
+	height: 0;
+	clear: both;
+	visibility: hidden;
+}
+.masterbar__oauth-client-wpcom {
+	padding: 3px 10px;
+	float: left;
+	cursor: pointer;
+	line-height: 32px;
+	margin: -3px 5px -3px -3px;
+	color: rgba(255, 255, 255, 0.8);
+}
+.masterbar__oauth-client-wpcom:hover {
+	color: #fff;
+}
+.masterbar__oauth-client-wpcom .gridicon {
+	margin-bottom: 5px;
+	margin-right: 5px;
+	vertical-align: middle;
+}
+.masterbar__oauth-client-wpcc-sign-in:hover {
+	background: rgba(255, 255, 255, 0.2);
+}
+.masterbar__oauth-client-wpcc-sign-in.active {
+	background: rgba(0, 0, 0, 0.9);
+	box-shadow: inset 0 1px 3px black, 0 1px 0 rgba(255, 255, 255, 0.2);
+}
+.masterbar__oauth-client-wpcc-sign-in.active .wpcom {
+	color: #fff;
+}
+
+.woo {
```

Comment:
That's the only thing woo related, shouldn't the stuff above it live in a separate file?